gpio/README.md

```diff
@@ -134,13 +145,15 @@ $ sudo pip3 install RPI.GPIO
 
 Parameter | Required | Restrictions | Purpose
 -|-|-|-
-`Class` | X | `btle_sensor.exec_actuator.ExecSensor` |
+`Class` | X | `gpio.rpi_gpio.RpiGpioActuator` |
 `Connection` | X | Comma separated list of Connections | Where the ON/OFF messages are published.
 `Level` | | DEBUG, INFO, WARNING, ERROR | When provided, sets the logging level for the sensor.
 `CommandSrc` | X | | Destination/openHAB switch item where commands are received, expects ON/OFF. If Toggle is set all messages trigger a toggle.
+`ToggleCommandSrc` | | | Destination/openHAB  string/datetime item where toggle commands are recieverd. This is intended to be used for direct connections to a sensor via the Short_Press-Dest/Long_Press-Dest parameter. Expects the string TOGGLE or the current time as java formated datetime value e.g. `2021-10-31T17:03:20.829945`. When one of those are recieved output of the actuator will get toggled e.g. from LOW to HIGH until further commands. The actuator expects changing datetime values, same ones are ignored. If the parameter `Toggle` is configured to TRUE this parameter is ignored.
```

Comment:
First of all with this code there are two ways to make the actuator toggle the output. Sending the string "TOOGLE" which works fine if only one connection is involved. If for some reason a sensor and an actuator are connected with more then one connection the sensor would send TOOGLE on each connection and the actuator would toggle each time since it doesn't know the command was intended only once. Therefore the gpio sensor will send a datetime to Short_Press-Dest and the actuator will only toggle if the datetime changes.
ToggleCommandSrc is intended to receive toggle commands either via the string "TOGGLE" or via datetime

Reply:
Maybe it would be more intuitive and require less work on the devices part  to make add a debounce parameter instead of sending date times. Then you can code in a time period after receiving a TOGGLE where you ignore all subsequent TOGGLE commands. 

I can think of lots of cases where needing to publish an ISO8601 string would be a really big deal and even where that's not a really big deal, it requires that the time be perfectly synchronized across all these potential sources of the command. If commands can indeed come from more than one device they almost certainly will never have the same time down to the msec level and in fact could be many minutes or hours apart in cases where there is no real time clock. On an Arduino and other microcontrollers the only clock is the number of msecs since the device started up.